docs/docs/guides/how-to-add-digest-to-email-notifications.md

```diff
@@ -0,0 +1,259 @@
+---
+sidebar_position: 2
+sidebar_label: How to Add Digest to Email Notifications
+---
+
+# How to Add Digest to Email Notifications
+
+## Introduction
+
+In this guide, you’ll learn how to add digest functionality to email notifications. But before exploring the actual code, let’s understand what a digest notification is and how it works.
+
+You can find the entire code(frontend as well as backend) for this app [here](https://github.com/novuhq/digest-email-app).
```

Comment:
Add a note section here just after this paragraph that refers to the digest for In-App notifications. Something like:

:::info
Check out how to add [digest to In-App Notifications in your app.](<insert-link-here)
:::

Reply:
okay

Reply:
done ✅